src/components/consent-drawer.js

```diff
@@ -1,22 +1,28 @@
+import md5 from 'md5';
 import { Button } from 'antd';
 import classNames from 'classnames';
 import Markdown from './markdown.js';
 import { useTranslation } from 'react-i18next';
+import { useLocale } from './locale-context.js';
 import { useService } from './container-context.js';
+import { useSettings } from './settings-context.js';
 import ClientConfig from '../bootstrap/client-config.js';
-import React, { useEffect, useState, useRef } from 'react';
 import CookieAlertIcon from './icons/general/cookie-alert-icon.js';
+import React, { useEffect, useState, useRef, useMemo } from 'react';
 import { getCookie, setLongLastingCookie } from '../common/cookie.js';
 
-const CONSENT_COOKIE_VERSION = 'm157c4noq4et';
-
+const generateCookieHash = consentTextInAllLanguages => {
+  return md5(JSON.stringify(consentTextInAllLanguages));
+};
 export default function ConsentDrawer() {
```

Comment:
Maybe we could have an empty line here, if you're up to it.

Reply:
🤦‍♀️  One of my plugins often makes me think empty lines are there where they aren't. Will add!

<img width="543" alt="Screen Shot 2022-04-15 at 18 59 02" src="https://user-images.githubusercontent.com/8189923/163599191-a5825962-647d-4ef6-b660-66df4a2d1d83.png">